tests/performance/framework/runner.go

```diff
@@ -0,0 +1,97 @@
+package framework
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"starknet-p2p-tests/config"
+	synthetic_node "starknet-p2p-tests/tools"
+)
+
+type TestFunc func(ctx context.Context, syntheticNode *synthetic_node.SyntheticNode) (time.Duration, error)
+
+func RunTest(b *testing.B, peerCount, requestsPerPeer int, rampUpTime, responseTimeout time.Duration, testFunc TestFunc) LatencyStats {
```

Comment:
Nit: Could we add a comment stating that RunTest starts a peer, runs the test, and collects the performance statistics?

Reply:
Comment added